.github/workflows/release-notes.yaml

```diff
@@ -0,0 +1,76 @@
+name: Create Changelog and Release
+# Trigger on tags matching the format: v0.0.0, V0.0.0, or 0.0.0
```

Comment:
Do we want to trigger on all? I think for consistency we should stick to a single format. Maybe switch to [Calendar Versioning](https://calver.org/) like HA itself did?